plugins/sarif-plugin/src/main/kotlin/ReportLintSarifPlugin.kt

```diff
@@ -0,0 +1,31 @@
+import com.android.build.api.dsl.LibraryExtension
+import com.android.build.gradle.internal.lint.AndroidLintTask
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.configure
+
+class ReportLintSarifPlugin : Plugin<Project> {
+
+    override fun apply(target: Project) {
+        target.plugins.withId("com.android.library") {
```

Comment:
What about the app/sandbox modules?

Reply:
IMHO do not see any value in linting these modules.